### PR TITLE
Remove bl wl service 

### PR DESCRIPTION
**Pull request by CAPtheorem**
Simplifies the stack by having the message-relayers directly call the `omgx_deployer` for filtering information.

### CODE REVIEW

**Review by sk-enya — APPROVED** [review]
All looks good.
We need to maintain the separate const files we can move all hard coding at one place.
Will do In another PR.

File: ops/docker-compose-omgx-services.yml [hunk 1]
```diff
@@ -34,25 +34,25 @@ services:
...
-    build:
-      context: ..
-      dockerfile: ./ops/docker/Dockerfile.omgx_bl-wl
-    environment:
-        OMGX_URL: http://omgx_deployer:8079/addresses.json
-        BL_WL_PORT: 9000
-        RETRIES: 70
-    ports:
-      - 9000:9000
+#   omgx_bl-wl:
```
Comment (InoMurko) [inline ops/docker-compose-omgx-services.yml:40]: pls delete if not needed

**Review by InoMurko — APPROVED** [review]

**Review by souradeep-das — APPROVED** [review]
looks great!

File: ops/docker-compose-omgx-services.yml (hunk 1 above)
Comment (CAPtheorem) [reply ops/docker-compose-omgx-services.yml:40]: Will do!

**Pull request merged**